spec/support/features/session_helper.rb

```diff
@@ -50,6 +50,7 @@ def sign_in_user(user = create(:user))
     end
 
     def sign_in_before_2fa(user = create(:user))
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
sure, works for me. I am sensitive to not including non-required test setup but it seems like in this case it's fine.